flowauth/frontend/cypress/integration/add_new_server.js

```diff
@@ -14,95 +14,96 @@ describe("Server management", function() {
   it("Add server name with space", function() {
     cy.get("#new").click();
     // adding username with space
-    cy.get("#name").type("Server ");
+    cy.get("#name").type("Server ", { force: true });
     //checking validation text
     cy.get("#name-helper-text").should(
       "have.text",
       "Server name may only contain letters, numbers and underscores."
     );
     cy.get("#name")
-      .type(" ")
-      .clear();
-    cy.get("#name").type("SERVER_TEST01");
+      .type(" ", { force: true })
+      .clear({ force: true });
+    cy.get("#name").type("SERVER_TEST01", { force: true });
     cy.contains("#name-helper-text").should("not.exist");
   });
   it("Add blank server name", function() {
     cy.get("#new").click();
     //adding blank username
     cy.get("#name")
-      .type(" ")
-      .clear();
+      .type(" ", { force: true })
+      .clear({ force: true });
     //checking validation text
     cy.get("#name-helper-text").should(
       "have.text",
       "Server name can not be blank."
     );
-    cy.get("#name").type("SERVER_TEST01");
+    cy.get("#name").type("SERVER_TEST01", { force: true });
     cy.contains("#name-helper-text").should("not.exist");
   });
   it("Add server name more than 120 characters", function() {
     cy.get("#new").click();
     //adding username
     cy.get("#name").type(
-      "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+      "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
```

Comment:
In a recent PR you had a nice concise way of writing this (something like `"a".repeat(121)` or similar). Do you want to change it here too?

Reply:
Ahh yes, in #864 
Yeah, might as well throw that in here as well.